_redirects

```diff
@@ -47,6 +47,9 @@
 /docs/products/clickhouse/howto/add-service-users                      /docs/products/clickhouse/howto/manage-users-roles
 /docs/products/clickhouse/howto/add-service-users.html                 /docs/products/clickhouse/howto/manage-users-roles.html
 /docs/products/clickhouse/sample-dataset.html                          /docs/products/clickhouse/howto/load-dataset.html
+/docs/products/flink/howto/create-job.html                             /docs/products/flink/howto/create-job/create-flink-applications.html 
+/docs/products/flink/concepts/flink-for-analysts                       /docs/products/flink/list-overview.html
+/docs/products/flink/concepts/flink-for-operators                      /docs/products/flink/list-overview.html
```

Comment:
Shouldn't we have entries also for the non `.html` version? e.g., on top of the ones you added, also 

```
/docs/products/flink/howto/create-job                            /docs/products/flink/howto/create-job/create-flink-applications 
/docs/products/flink/concepts/flink-for-analysts.html                       /docs/products/flink/list-overview.html
/docs/products/flink/concepts/flink-for-operators.html                      /docs/products/flink/list-overview.html
```

Reply:
added ` .html` to all files.